hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -297,21 +295,15 @@ private FlinkOptions() {
       .booleanType()
       .defaultValue(false)// default read as batch
       .withDescription("Whether to skip compaction instants and avoid reading compacted base files for streaming read to improve read performance.\n"
-          + "There are two cases that this option can be used to avoid reading duplicates:\n"
-          + "1) you are definitely sure that the consumer reads [faster than/completes before] any compaction instants "
-          + "when " + HoodieCompactionConfig.PRESERVE_COMMIT_METADATA.key() + " is set to false.\n"
-          + "2) changelog mode is enabled, this option is a solution to keep data integrity");
+          + "This option can be used to avoid reading duplicates when changelog mode is enabled, it is a solution to keep data integrity\n");
```

Comment:
@danny0405 compaction always preserves the commit metadata by default and now we make it the only behavior to avoid corruption.  I assume this is ok for Flink too.  Could you confirm it?